### PR TITLE
Pass options to `prepend-http`

### PR DESCRIPTION
**Pull request by gillesdemey**
Allows passing options to `prepend-http` to allow the HTTPS protocol to be set by default.

### CODE REVIEW

**Review by SamVerschueren — CHANGES_REQUESTED** [review]
Can you add docs and a test?

**Comment by gillesdemey** [comment]
Will get on that :)

**Comment by gillesdemey** [comment]
It seems that Travis tests run on an ancient version of Node and complains about ES6 syntax, thoughts on how to proceed?

**Comment by SamVerschueren** [comment]
Added a PR to drop support for ancient Node versions. So let's wait until Sindre reviewed and merged that one. Will make things easier :).

**Comment by gillesdemey** [comment]
rebased from master, PTAL :)

**Review by SamVerschueren — CHANGES_REQUESTED** [review]
I added an API section, can you add it there as well in the same fashion as https://github.com/sindresorhus/prepend-http#prependhttpurl-options.

**Comment by gillesdemey** [comment]
Ah I missed that, added it to the API section with a link to `prepend-http`, or should we copy over the documentation instead?

**Comment by SamVerschueren** [comment]
> or should we copy over the documentation instead?

I'm ok with what we have now. I'll let @sindresorhus decide :).

Thanks for the PR!

**Review by SamVerschueren — APPROVED** [review]

**Pull request merged**
